libs/wingc/src/jsify.rs

```diff
@@ -145,6 +145,33 @@ impl JSifier {
 			}
 		}
 	}
+	// note: Globals are emitted here and wrapped in "{{ ... }}" blocks. Wrapping makes these emissions, actual
```

Comment:
Why double `{{`?

Reply:
[It's Rust syntax](https://doc.rust-lang.org/std/fmt/index.html#escaping). It's how a literal `{` is escaped in formatted strings.